kano_feedback/kano_feedback_cli.py

```diff
@@ -15,13 +17,28 @@
 from kano_feedback.return_codes import RC
 
 
+def _check_can_create_file_flag(flag_path):
+    """TODO"""
+    success = False
+    try:
+        if touch(flag_path):
+            os.remove(flag_path)
```

Comment:
If `remove()` fails at this point for some reason will this cause the function to always fail? Presumably not because `touch()` presumably is a no-op in the case where the file already exists (or just updates the timestamp) and if the file is still there the next time it is run then the `remove()` will test for the permissions anyway.